Backend/MagentoAPI.php

```diff
@@ -81,6 +82,7 @@ public function setValue($key, $value) {
             $keyValueModel->setData(InstallSchema::CLOUDFLARE_DATA_TABLE_VALUE_COLUMN, $value);
             $keyValueModel->save();
         }
+        return true;
```

Comment:
https://github.com/cloudflare/cloudflare-plugin-backend/blob/master/src/API/AbstractPluginActions.php#L154

needs a return value or the PATCH /settings fails.